src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  margin-bottom: 46px;
+  background-color: #fff;
+  color: #060b35;
+  border: 1px solid #f3f3f3;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
```

Comment:
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)

Reply:
organize your styles pls

Reply:
I have already oganized my styles like here https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
![111](https://user-images.githubusercontent.com/52597158/222428849-8e28cb4a-e375-4326-a868-f58223e09252.png)